src/scenes/home/press/pressLinks/pressLinks.css

```diff
@@ -0,0 +1,16 @@
+.logos a {
+  color: #249cbc;
+  text-decoration: none;
+}
+
+@media screen and (min-width: 415px) and (max-width: 1280px) {
+  .logos > .imgBox {
+    width: 350px;
```

Comment:
I don't see any elements with the class `.imgBox` in the markup - so is there any need for this CSS rule?